docs/digger/admin/use-http-proxy.asciidoc

```diff
@@ -1,3 +1,5 @@
+== HTTP Proxy Support
+
 === Configuration Options
 
 It is possible to run AeroGear Digger behind a HTTP proxy server. To configure the proxy server, you need to add the following Ansible Variables:
```

Comment:
although we do provide a [role](https://github.com/fheng/rhmap-ansible/blob/master/roles/proxy-setup/tasks/main.yml) to generate the proxy_url in the rhmap-installer. We could look at moving that role to the digger-installer also? In fact right now only the buildfarm playbook [relies on that role](https://github.com/fheng/rhmap-ansible/search?utf8=%E2%9C%93&q=proxy-setup&type=) so it would be no problem to move it.